backend/src/main/java/com/empire/World.java

```diff
@@ -490,7 +490,7 @@ public double sum() {
 		int inspires;
 		HashMap<Army, Character> leaders = new HashMap<>();
 		HashMap<String, Double> pirateThreatSources = new HashMap<>();
-		HashMap<Region, ArrayList<Character>> governors = new HashMap<>();
+		HashMap<Region, Character> governors = new HashMap<>();
```

Comment:
I'm a broken record here but, these could all be Maps and Sets.  At least for class members I feel this is actually important.

Reply:
Yeah, I agree, for non-private members especially. That being said, I'd rather avoid a merge conflict and not fix it in this PR, which is only getting rid of the ArrayList. I know I'm inconsistent on that. >_>